Hometask_2/Hometask_2.py

```diff
@@ -0,0 +1,40 @@
+from collections import defaultdict
+import random
+import string
+
+
+# Create the list dictionaries from 2 to 10.
+def create_dicts():
+    output = []  # Dicts list
+    for dict_number in range(random.randint(2, 10)):  # Dictionary counter
+        dictionary = {}
+        for elem_number in range(random.randint(1, 26)):  # Elements counter
+            # Key(random letter), value(random number from 0 to 100)
+            dictionary[random.choice(string.ascii_lowercase)] = random.randint(0, 100)
+        output.append(dictionary)  # Append every dictionary to the list
+    return output
+
+
+output = defaultdict(lambda: float('-inf'))
+
+list_of_dicts = create_dicts()
+
+
+for cntr in range(0, len(list_of_dicts)):  # Create a counter for later adding of '_number_of_dictionary' for the same keys
+    for key, value in list_of_dicts[cntr].items():
```

Comment:
`
for cntr, one_dict in enumerate(list_of_dicts, start=1):
    for key, value in one_dict.items():
        print(cntr, key, value)
`
'enumerate' is nice function that make your code more readable.